Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax = docker/dockerfile:1.4
+FROM public.ecr.aws/docker/library/fedora:38
+
+# install necessary cloud-server packages
+RUN dnf group install -y cloud-server-environment --exclude=plymouth* \
+    --exclude=geolite* \
+    --exclude=firewalld* \
+    --exclude=grub* \
+    --exclude=dracut* \
+    --exclude=shim-*
+
+RUN systemctl enable cloud-init cloud-init-local cloud-config cloud-final
```

Comment:
Just for posterity's sake: we're not 100% sold on the need for cloud-init (at least for the WSL2 rootfs) yet.

Reply:
Thanks yea... let me include that in both PR description and commit msg as well to preserve